docs/sections/user_guide/cli/drivers/chgres_cube.rst

```diff
@@ -93,6 +93,9 @@ Its contents are described in depth in section :ref:`chgres_cube_yaml`. Each of
 
      $ uw chgres_cube run --config-file config.yaml --cycle 2023-12-15T18 --batch --dry-run
 
+
+.. include:: ../../../../shared/key_path.rst
```

Comment:
This looks good to me in the rendered docs. If no one else disagrees, I think we could add this line to the other driver `.rst` files, since they all inherit the `--key-path` functionality.